taskDev4/src/textHandler/ConsoleReader.java

```diff
@@ -0,0 +1,60 @@
+package textHandler;
+
+import java.io.*;
+import jdk.nashorn.internal.runtime.regexp.joni.Regex;
+
+/**
+ * Works with console.
+ */
+public class ConsoleReader {
+  private static final String NOTHING_SYMB = "";
+  private static final String ENTER_TEXT_ASK = "Enter text: ";
+  private static final String ENTER_MORE_TEXT_QUESTION = "Do you want enter more text? (yes/no)";
+  private static final String NO_ANSWER = "no";
+  private static final String YES_ANSWER = "yes";
+
+  /**
+   * Read text from console.
+   *
+   * @return text from console.
+   * @throws IOException for BufferedReader.
+   */
+  public String read() throws IOException {
+    BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+    String regex = "[\\d~`@#$%^&()_|№*-+]+"; // Ignored symbols
+
+    System.out.println(ENTER_TEXT_ASK);
+
+    return reader.readLine().replaceAll(regex,NOTHING_SYMB);
+  }
+
+  /**
+   * Check conditions for cycle.
+   *
+   * @return true, if it needs to repeat.
+   * @throws IOException for BufferedReader input.
+   */
+  public boolean needRepeat() throws IOException {
```

Comment:
process IOException inside cycle to give opportunity to repeat if something goes wrong